tutorials/applications/QAOA and QAA to solve a MIS problem.ipynb

```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>I think you should add a paragraph saying we start by trying to use QAOA to solve the problem (and maybe some motivation why).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>